git-validation barfs on gpg signature verification output of git log

We're having a repeated problem using this tool in containers/buildah where exit-codes are being returned without any corresponding error message.  For example:

```
+ exec git-validation -v -run DCO,short-subject -range 2b32b824f81b4397f3082ed284d26f7e04b380a8..HEAD
time="2020-09-16T08:30:49-04:00" level=error msg="mm[git] cmd: \"git --no-pager log --pretty=format:%H 2b32b824f81b4397f3082ed284d26f7e04b380a8..HEAD\""
2020/09/16 08:30:49 exit status 128
```

If I execute that quoted git command locally in the repo., the tool `exit(0)`.  These are terribly difficult to debug...any assistance you could provide (even a clue where to look) would be extremely helpful.

Update: I was able to reproduce this, and the problem appears to be caused by gpg-signed commits:

$ make validate GITVALIDATE_FLAGS="-v -D"
make -C tests/tools
make[1]: Entering directory '/home/cevich/devel/buildah/tests/tools'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/cevich/devel/buildah/tests/tools'
+ exec git-validation -q -run DCO,short-subject -range 2b32b824f81b4397f3082ed284d26f7e04b380a8..HEAD -v -D
2020/09/16 10:00:17 []validate.Rule{validate.Rule{Name:"DCO", Value:"", Description:"makes sure the commits are signed", Run:(func(validate.Rule, git.CommitEntry) validate.Result)(0x4ed1c0), Default:true}, validate.Rule{Name:"short-subject", Value:"", Description:"commit subjects are strictly less than 90 (github ellipsis length)", Run:(func(validate.Rule, git.CommitEntry) validate.Result)(0x4eda60), Default:true}}
INFO[0000] [git] cmd: "git --no-pager log --pretty=format:%H 2b32b824f81b4397f3082ed284d26f7e04b380a8..HEAD"
INFO[0000] [git] cmd: "git --no-pager log -1 --pretty=format:%GS gpg: Signature made Wed 16 Sep 2020 08:26:07 AM EDT"
fatal: invalid object name 'gpg'.
ERRO[0000] [git] cmd: "git --no-pager log -1 --pretty=format:%GS gpg: Signature made Wed 16 Sep 2020 08:26:07 AM EDT"
2020/09/16 10:00:17 exit status 128
make: *** [Makefile:97: validate] Error 1

--- a/git/commits.go
+++ b/git/commits.go
@@ -15,7 +15,7 @@ import (
 // If commitrange is a single commit, all ancestor commits up through the hash provided.
 // If commitrange is an empty commit range, then nil is returned.
 func Commits(commitrange string) ([]CommitEntry, error) {
-	cmdArgs := []string{"git", "--no-pager", "log", `--pretty=format:%H`, commitrange}
+	cmdArgs := []string{"git", "rev-list", commitrange}
 	if debug() {
 		logrus.Infof("[git] cmd: %q", strings.Join(cmdArgs, " "))
 	}

